python/graphstorm/inference/ep_infer.py

```diff
@@ -27,15 +27,21 @@
 from ..utils import sys_tracker, get_rank, barrier
 
 class GSgnnEdgePredictionInferrer(GSInferrer):
-    """ Edge classification/regression inferrer.
+    """ Inferrer for edge prediction tasks.
 
-    This is a high-level inferrer wrapper that can be used directly
-    to do edge classification/regression model inference.
+    ``GSgnnEdgePredictionInferrer`` inherits from the ``GSInferrer`` and use the functions
+    provided by ``GSInferrer`` to define the ``infer()`` method that performs three works:
```

Comment:
Same here, let's avoid implementation details in the docstrings

Reply:
Have updated all Trainers and Inferrers class doc string.